data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/reconciler/impl/ResourcesReconcilerMessageHandler.java

```diff
@@ -35,12 +36,15 @@ public class ResourcesReconcilerMessageHandler implements Handler<Message<Object
   private static final Logger logger = LoggerFactory.getLogger(ResourcesReconcilerMessageHandler.class);
 
   public final static String ADDRESS = "resourcesreconciler.core";
+  public static final int RECONCILE_TIMEOUT = 10000;
 
+  private final Vertx vertx;
   private final ResourcesReconciler resourcesReconciler;
   private final AtomicBoolean reconciling;
   private final AtomicReference<DataPlaneContract.Contract> last;
 
-  public ResourcesReconcilerMessageHandler(final ResourcesReconciler resourcesReconciler) {
+  public ResourcesReconcilerMessageHandler(final Vertx vertx, final ResourcesReconciler resourcesReconciler) {
+    this.vertx = vertx;
```

Comment:
for the new reconcilation timeout we need to pass the vertx instance here?

Reply:
Correct